cornice/cors.py

```diff
@@ -125,7 +125,7 @@ def apply_cors_post_request(service, request, response):
             'Access-Control-Allow-Credentials' not in response.headers):
         response.headers['Access-Control-Allow-Credentials'] = 'true'
 
-    if request.method is not 'OPTIONS':
+    if request.method != 'OPTIONS':
```

Comment:
Okay, I just reproduced, this is weird because not reproductible each time.
``` .tox/py27/bin/nosetests -s cornice.tests.test_cors:TestCORS.test_preflight_does_not_expose_headers
> /home/alexis/dev/github.com/cornice/cornice/cors.py(129)apply_cors_post_request()
-> if request.method != 'OPTIONS':
(Pdb) request.method
'OPTIONS'
(Pdb) request.method is not 'OPTIONS'
True
(Pdb) request.method != 'OPTIONS'
False
```